tox.ini

```diff
@@ -0,0 +1,33 @@
+[tox]
+envlist =
+    flake8,
+    py27,
+    py36,
+    py37,
+    coverage,
+
+skip_missing_interpreters = False
+
+[testenv]
+usedevelop = true
+commands =
+    zope-testrunner --all --test-path=src {posargs:-vc}
+extras = test
+deps =
+    zope.testrunner
+
+[testenv:coverage]
+basepython = python3.6
+deps =
+    zope.testrunner
+    coverage
+commands =
+    coverage run -m zope.testrunner --all --test-path=src {posargs:-vc}
+    coverage html
+    coverage report --fail-under=71
+
+[testenv:flake8]
+basepython = python2.7
```

Comment:
Is there a reason we stick with python2.7 here instead of moving on to python3.6 as in `coverage`?